certbot/lock.py

```diff
@@ -80,7 +80,9 @@ def _try_lock(self, fd):
                 logger.debug(
                     "A lock on %s is held by another process.", self._path)
                 raise errors.LockError(
-                    "Another instance of Certbot is already running.")
+                    "Another instance of Certbot is already running.\n\n"
+                    "If you really need to run two Certbot instances at once, provide a "
+                    "distinct --config-dir, --work-dir, and --logs-dir to each of them.")
```

Comment:
Exception messages that include newlines can make logs harder to read/grep. It can be especially annoying when using log parsing/monitoring tools. I'm not sure it's an issue for certbot logs.

A different way to write this error message would be: "Another instance of Certbot is already running using the same --config-dir, work-dir, or --logs-dir." This isn't quite as helpful, but it's a lot more concise and I think it's enough to make the solution clear.